cosmos/templates/news/news_create.html

```diff
@@ -5,15 +5,18 @@
 {% block content %}
 
 <div class="container">
-    <div class="mx-auto w-50">
-        <form method="post" enctype="multipart/form-data" id="form-container">
+    <div class="mx-auto w-50 bg-cosmos-primary rounded">
+        <form method="post" enctype="multipart/form-data" id="form-container" class="p-4 overflow-hidden">
+            
+            <h2 class="text-white">Create news article</h2>
+            
             {% csrf_token %}
             {{ form.media }}
             {% crispy form %}
```

Comment:
same issue as in gmm_update.html
![image](https://user-images.githubusercontent.com/19555626/127763687-dfcfa201-c959-4d85-8a1f-0eb7c67cf6e4.png)